tests/link/pytorch/test_sort.py

```diff
@@ -7,17 +7,12 @@
 from tests.link.pytorch.test_basic import compare_pytorch_and_py
 
 
+@pytest.mark.xfail(reason="Reshape not implemented")
 @pytest.mark.parametrize("axis", [0, 1, None])
```

Comment:
When only some of the parametrized conditions fail, you should use `pytest.param` to mark that specifically as xfailing. The others are working and shouldn't be marked. There are some examples in the tests somewhere